tmt/steps/discover/shell.py

```diff
@@ -225,8 +232,10 @@ class DiscoverShell(tmt.steps.discover.DiscoverPlugin[DiscoverShellData]):
     Use provided list of shell script tests.
 
     List of test cases to be executed can be defined manually directly
-    in the plan as a list of dictionaries containing test name, actual
-    test script and optionally a path to the test. Example config:
+    in the plan as a list of dictionaries containing test ``name``, actual
+    ``test`` script and optionally the ``duration`` or a ``path`` to the test.
```

Comment:
We support any test metadata keys here, right? So, I guess, we could mention `duration` and `path` just as example of other metadata?

Reply:
Ohhh I get it, yeah that sounds good to me :)